### ISSUE TITLE
Training stopping on epoch 19

### ISSUE THREAD

**Issue by AdiPathak97** [tpvt99/character-recognition-cnn#3]
![cnn_hang](https://user-images.githubusercontent.com/36631927/47414050-a3025600-d78d-11e8-9693-11d3f2fefbf9.PNG)
Training has stopped here and not progressing since the last 30 minutes. If I force escape, test.py doesn't run due to a file not being generated due to incomplete training.

**Comment by tpvt99** [comment]
hi. have you solved your problem yet?

**Comment by AdiPathak97** [comment]
The issue specified in the screenshot was a rather silly oversight on my part, where I didn't close the plot window, rather just minimized it. However, once I closed the window, cnn.py finished execution, but threw up another error saying that 'my-model.ckpt cannot be saved, cannot find specified directory' or something to that effect. I don't have a screenshot of that error. 

The cause of this error was that 'my-model.ckpt' file wasn't being generated in the current directory as specified in line 185 of cnn.py. This is the following line : 
`saver.save(sess, 'my-model.ckpt')`

I resolved this by editing this line to this :
`saver.save(sess, './my-model.ckpt')`
This saves generated file into root directory of the folder.
